rpcs3/rpcs3qt/log_frame.h

```diff
@@ -72,7 +75,10 @@ private Q_SLOTS:
 	QAction* m_noticeAct;
 	QAction* m_traceAct;
 
-	QAction* m_stackAct;
+	QAction* m_stackAct_log;
+	QAction* m_stackAct_tty;
+	
```

Comment:
What's with all the empty lines?
They are not needed.

Reply:
It's not showing as outdated but I did remove the empty lines.

Reply:
It looks like there's a change from empty to empty. Just wondering why it appears as line change

Reply:
I probably typed it weirdly? It displays like this for me:
```
QAction* m_traceAct;

QAction* m_stackAct_log;
QAction* m_stackAct_tty;
	
QAction* m_TTYAct;

QActionGroup* m_tty_channel_acts;
```

Reply:
You added a tab at the beginning of the empty line.